md/script/evalCodeBlocks.ts

```diff
@@ -56,15 +57,13 @@ export async function evalCodeBlocks(
     const iconLength = icon === "skip" ? 4 : 1;
     const langLength = String(lang).length;
 
+    const prefix = `${colorIcon} ${details.lang} `;
+    const spacesInPrefix = 2;
+
     const inlineCode = code.trim().replace(/\s+/g, " ");
-    const firstChars = inlineCode.slice(
-      0,
-      consoleWidth - langLength - iconLength - 5,
-    );
-
-    const message = firstChars.length < inlineCode.length
-      ? `${colorIcon} ${details.lang} ${firstChars}...`
-      : `${colorIcon} ${details.lang} ${firstChars}`;
+    const maxLength = consoleWidth - langLength - iconLength - spacesInPrefix;
+
+    const message = `${prefix}${elideEnd(inlineCode, { maxLength })}`;
```

Comment:
Result now uses 2 fewer characters through use of length-1 `…` instead of 3 periods.

<img width="353" alt="Screenshot 2023-05-26 at 12 57 13 PM" src="https://github.com/pskfyi/handy/assets/47710574/76861cf6-6364-449a-99a0-1746089284fa">